src/features/lightspeed/inlineSuggestions.ts

```diff
@@ -835,5 +823,61 @@ export async function inlineSuggestionUserActionHandler(
     inlineSuggestionFeedbackPayload,
     lightSpeedManager.orgTelemetryOptOut
   );
+  resetSuggestionData();
+}
+
+function inlineSuggestionPending(): boolean {
+  if (vscode.window.activeTextEditor?.document.languageId !== "ansible") {
+    return false;
+  }
+  const editor = vscode.window.activeTextEditor;
+  if (!editor) {
+    return false;
+  }
+  if (!inlineSuggestionData["suggestionId"]) {
+    return false;
+  }
+  return true;
+}
+
+function resetSuggestionData(): void {
   inlineSuggestionData = {};
+  insertTexts = [];
+}
+
+export async function textDocumentChangeHandler(
```

Comment:
I think the function name is a bit too generic and we can be more explicit about its relationship with the feedback mechanism.